test/Microsoft.Framework.Logging.Test/LoggerFactoryExtensionsTest.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Framework.Logging.Test
     public class LoggerFactoryExtensionsTest
     {
 #if ASPNET50
```

Comment:
Wondering - any reason for this `#ifdef`? Does this stuff not work on any other platform?